src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
```

Comment:
BTW you don't need wrapper here. You may move this class to each block with modifier `stars--{number}`

Reply:
1. Okay, maybe I didn't explain it that way. It is better to remove that wrapper, it is not needed, all the blocks will be under each other anyway.
2. According to BEM, it is not very readable when the block modifier is embedded in other sub-blocks. It is better to use the star class instead of the row class